docs/.vitepress/vitepress/components/vp-app.vue

```diff
@@ -18,6 +20,37 @@ useToggleWidgets(isSidebarOpen, () => {
     toggleSidebar(false)
   }
 })
+
+onMounted(() => {
+  window.addEventListener(
+    'click',
+    (e) => {
+      const link = e.target.closest('a')
+      if (link) {
```

Comment:
Why not use `if(!link) return` to reduce nested

Reply:
Done